template-processors/applier/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM quay.io/kohlstechnology/eunomia-base:v0.0.1
```

Comment:
it would be better to base this on `eunomia-ocp-templates`, so that we keep the versions consistent.

Reply:
currently the `oc` client in the ocp templates image doesn't work, so it doesn't seem like a reliable base.

Reply:
You could just fix that one as part of your PR ;-)